cfgov/jinja2/v1/_includes/molecules/image-text-50-50.html

```diff
@@ -12,63 +12,32 @@
    See [GHE]/flapjack/Modules-V1/wiki/50-50-Image-&-Text
 
    value:                An object with the following options for value.
-   value.heading:          A string containing title of module.
+   value.heading:        A string containing title of module.
    value.image:          A image object containing the URL & alt text for screenreaders
-                            of the image to be placed adjacent to the text.
+                         of the image to be placed adjacent to the text.
    value.is_widescreen:  Boolean if the image should be widescreen ratio.
```

Comment:
This has to wait till we have the backend code to match (see ##Todo section in PR) . Currently it's set to widescreen w/ a default to 4:3, but we're no longer setting full width as bg images, only square, so it made more sense to limit this to square in the info-unit itself. I'm adding an inline `TODO` for it now so it's not lost.